jobs/onceoff/migrate_legacy_data.rb

```diff
@@ -0,0 +1,20 @@
+module Jobs
+
+  class MigrateLegacyData < Jobs::Onceoff
+
+    PLUGIN_NAME = ::DiscourseSlack.plugin_name.freeze
```

Comment:
I created just for my convenient. Else I have to call it every time like `::DiscourseSlack.plugin_name`. But `PLUGIN_NAME` is shorter. Is it bad practice. I don't know.